frontend/src/metabase/admin/permissions/components/PermissionsEditor.jsx

```diff
@@ -29,27 +29,31 @@ const PermissionsEditor = ({
 }) => {
   const saveButton = (
     <Confirm
-      title={t`Save permissions?`}
+      title="Save permissions?"
```

Comment:
Why remove the i18n tags here and in the rest of the file?

Reply:
Yikes. That particular change wasn't intentional. Might've been the result of me checking out the file from origin/master after I messed with it?